cpp/example_code/sns/README.md

```diff
@@ -58,20 +54,28 @@ Code excerpts that show you how to call individual service functions.
 * [Get the settings for sending SMS messages](get_sms_type.cpp#L23) (`GetSMSAttributes`)
 * [List the subscribers of a topic](list_subscriptions.cpp#L23) (`ListSubscriptions`)
 * [List topics](list_topics.cpp#L23) (`ListTopics`)
+* [Publish a message with an attribute](../cross-service/topics_and_queues/messaging_with_topics_and_queues.cpp#L509) (`Publish`)
 * [Publish an SMS text message](publish_sms.cpp#L23) (`Publish`)
 * [Publish to a topic](publish_to_topic.cpp#L23) (`Publish`)
 * [Set the default settings for sending SMS messages](set_sms_type.cpp#L23) (`SetSmsAttributes`)
 * [Subscribe a Lambda function to a topic](subscribe_lambda.cpp#L23) (`Subscribe`)
 * [Subscribe a mobile application to a topic](subscribe_app.cpp#L23) (`Subscribe`)
-* [Subscribe an Amazon SQS queue to a topic](../cross-service/topics_and_queues/messaging_with_topics_and_queues.cpp#L809) (`Subscribe`)
+* [Subscribe an SQS queue to a topic](../cross-service/topics_and_queues/messaging_with_topics_and_queues.cpp#L817) (`Subscribe`)
 * [Subscribe an email address to a topic](subscribe_email.cpp#L18) (`Subscribe`)
-* [Subscribe with a filter to a topic](../cross-service/topics_and_queues/messaging_with_topics_and_queues.cpp#L809) (`Subscribe`)
+* [Subscribe with a filter to a topic](../cross-service/topics_and_queues/messaging_with_topics_and_queues.cpp#L424) (`Subscribe`)
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple
+functions within the same service.
+
+* [Publish messages to queues](../cross-service/topics_and_queues/messaging_with_topics_and_queues.cpp)
 
 ### Cross-service examples
 
 Sample applications that work across multiple AWS services.
 
-* [Publish Amazon SNS messages to Amazon SQS queues](../../example_code/cross-service/topics_and_queues) 
+* [Create a serverless application to manage photos](../../example_code/cross-service/photo_asset_manager) 
```

Comment:
Is this correct? We are taking out the topics and queues and replacing it with a link to PAM? Or is it because the scenario is listed below instead?

Reply:
Yes, I did not update the README when I moved the workflow from cross-service to scenario.  So this is the new WRITEME output with SNS workflow added manually until an issue is fixed.